Cargo.toml

```diff
@@ -98,3 +98,6 @@ codegen-units = 256
 [profile.release-lto]
 inherits = "release"
 lto = true
+
+[patch.crates-io]
+cargo-generate = { git = 'https://github.com/cargo-generate/cargo-generate.git' }
```

Comment:
is there a reason to dep on git?  if so, lets the lockdown to actual git hash

Reply:
`cargo-generate`'s latest published crate has a dependency on tempfile uses `tempfile = "~3.3` which uses the vulnerable `remove_dir_all`. Will add hash